website/src/IndexPage.tsx

```diff
@@ -192,219 +196,30 @@ export function Products() {
 }
 
 export function SupportedNetworks() {
-  const { t } = useI18n()
+  const { supportedNetworks } = useData() as { supportedNetworks: Awaited<ReturnType<typeof getSupportedNetworks>> }
```

Comment:
is there no automatic type inference? 🤔

Reply:
Hmm, how would it know? I checked and `useData()` doesn't accept a generic type either.

Reply:
@dimaMachina maybe we can add support for generic?

Reply:
This hook will be removed in near Nextra 4 in favor of server components